fendermint/app/src/app.rs

```diff
@@ -180,6 +184,9 @@ where
         chain_env: ChainEnv,
         snapshots: Option<SnapshotClient>,
     ) -> Result<Self> {
+        let upgrade_scheduler = load_upgrade_scheduler();
+        upgrade_scheduler.validate()?;
```

Comment:
Shouldn't it already return a validated scheduler, or otherwise make sure we can't forget to call it?